libr/core/cmd_print.c

```diff
@@ -437,7 +437,7 @@ R_API int r_core_process_input_pade(RCore *core, const char *input, char** hex,
 
 static void print_format_help(RCore *core) {
 	const char* help_msg[] = {
-	"pf:", "pf[.k[.f[=v]]|[ v]]|[n]|[0][ [sz] fmt] [a0 a1 ...]", "",
+	"pf:", "pf[.k[.f[=v]]|[v]]|[n]|[0|cnt][fmt] [a0 a1 ...]", "",
```

Comment:
Done. Maybe we should dump the 5-level deep nested switches and go for a real parser with BNF, and autogen usage always correct and non-ambigouus: